src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -35,119 +35,120 @@
+                           dest-conf-namespace (.-value (.getDOMNode (@refs "destinationNamespace")))
+                           post-data (create-post-data
+                                       selected-conf-name
+                                       selected-conf-namespace
+                                       selected-conf-snapshot-id
+                                       dest-conf-name
+                                       dest-conf-namespace)]
+                       (utils/ajax-orch
+                         (paths/copy-method-config-to-workspace-path workspace-id)
+                         {:headers {"Content-Type" "application/json"}
+                          :canned-response {:responseText
+                                            (utils/->json-string (create-mock-methodconfs-import))
+                                            :status 200
+                                            :delay-ms (rand-int 2000)}
+                          :method :post
+                          :data (utils/->json-string post-data)
+                          :on-done (fn [{:keys [success? xhr]}]
+                                     (if success?
+                                       ((:on-import props))
+                                       (js/alert (str "Import Error: "  (.-responseText xhr)))))})))}]]))})
 
+; Render out the table component - just need the actual configurations to list and a handler to call when
+; the user selects a configuration
+(react/defc ConfigurationsTable
+  {:render
+   (fn [{:keys [props]}]
+     [:div {}
+      (formatted-header "Select A Method Configuration For Import")
+      [table/Table
+       {:columns [{:header "Name" :starting-width 200 :filter-by #(% "name") :sort-by #(% "name")
+                   :content-renderer
+                   (fn [row-index conf]
+                     [:a
+                      {:onClick
+                       (fn []
+                         ((:on-config-selected props) conf))
+                       :href "javascript:;"
+                       :style {:color (:button-blue style/colors) :textDecoration "none"}}
+                      (conf "name")])}
+                  {:header "Namespace" :starting-width 200 :sort-by :value}
+                  {:header "Snapshot Id" :starting-width 100 :sort-by :value}
+                  {:header "Synopsis" :starting-width 160 :sort-by :value}
+                  {:header "Create Date" :starting-width 210 :sort-by :value}
+                  {:header "Owner" :starting-width 290 :sort-by :value}]
+        :data (map
+                (fn [config]
+                  [config
+                   (config "namespace")
+                   (config "snapshotId")
+                   (config "synopsis")
+                   (config "createDate")
+                   (config "owner")])
+                (:method-configs props))}]])})
 
 (react/defc ImportWorkspaceMethodsConfigurationsList
```

Comment:
This component really is both the form and the table depending on state, so maybe something like `ModalContent` or even just a generic `Page`.